java/adapter/jdbc/src/test/java/org/apache/arrow/adapter/jdbc/ResultSetUtilityTest.java

```diff
@@ -50,6 +50,7 @@ public void testZeroRowResultSet() throws Exception {
         assertNotNull("VectorSchemaRoot from first next() result should never be null", root);
         assertEquals("VectorSchemaRoot from empty ResultSet should have zero rows", 0, root.getRowCount());
         assertFalse("hasNext() should return false on empty ResultSets after initial next() call", iter.hasNext());
+        root.close();
```

Comment:
This is needed to release initial offset buffer for empty array.

Reply:
Perhaps we can use a `try` instead of calling `close` explicitly?